ui/src/components/VoxPrompts.js

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {LitElement, html} from 'lit-element';
+
+import promptApi from '../utils/PromptApi';
+
+export class VoxPrompts extends LitElement {
+    static get properties() {
+        return {
+            prompt: {type: String},
+        };
+    }
+    constructor() {
+        super();
+        this.getNewPrompt();
+    }
+
+    getNewPrompt() {
+        promptApi
+            .getNewPrompt()
+            .then((nextPrompt) => (this.prompt = nextPrompt));
```

Comment:
getNewPrompt  asynchronously sets this.prompt. It's possible that the first render occurs before this.prompt has been set. You'll need to set some sort of ready state to true after the prompt is retrieved and only render the prompt elements when the prompt is ready.
```
constructor() {
...
this.ready
...
}

getNewPrompt() {
...
this.prompt = prompt;
this.ready = false;
...
}
```

You should also catch any errors from fetching and render something based on the error.

Ideally you should always handle (render something that makes sense and tells the user something useful for) the following states for your UI if you have a fetch request:
NOT_ASKED, LOADING, SUCESS, FAILURE
See graphic here: https://elmprogramming.com/remote-data.html